config.json

```diff
@@ -1,428 +1,434 @@
+      "unlocked_by": null,
+      "uuid": "fab94f5a-d4da-4a18-a70a-b27c80b14851"
     },
     {
-      "uuid": "a384c32c-d8db-4877-816b-44de7c3ed324",
-      "slug": "beer-song",
       "core": false,
-      "unlocked_by": null,
       "difficulty": 7,
+      "slug": "beer-song",
       "topics": [
         "strings",
-        "text formatting"
-      ]
+        "text_formatting"
+      ],
+      "unlocked_by": null,
+      "uuid": "a384c32c-d8db-4877-816b-44de7c3ed324"
     },
     {
-      "uuid": "675b0a5c-0587-4dc5-8004-1f82769a2ddd",
-      "slug": "nth-prime",
       "core": false,
-      "unlocked_by": null,
       "difficulty": 5,
+      "slug": "nth-prime",
       "topics": [
         "mathematics"
-      ]
+      ],
+      "unlocked_by": null,
+      "uuid": "675b0a5c-0587-4dc5-8004-1f82769a2ddd"
     },
     {
-      "uuid": "c2e8e3fc-a39c-434b-87dd-9524636aa804",
-      "slug": "hexadecimal",
       "core": false,
-      "unlocked_by": null,
       "difficulty": 3,
+      "slug": "hexadecimal",
       "topics": [
-        "strings",
-        "mathematics"
-      ]
+        "mathematics",
+        "strings"
+      ],
+      "unlocked_by": null,
+      "uuid": "c2e8e3fc-a39c-434b-87dd-9524636aa804"
     },
     {
-      "uuid": "0d5e4c9e-4351-41b1-9154-ee900ad0e60e",
-      "slug": "queen-attack",
       "core": false,
-      "unlocked_by": null,
       "difficulty": 5,
+      "slug": "queen-attack",
       "topics": [
-        "logic",
-        "control-flow (loops)"
-      ]
+        "control_flow_loops",
+        "logic"
+      ],
+      "unlocked_by": null,
+      "uuid": "0d5e4c9e-4351-41b1-9154-ee900ad0e60e"
     },
     {
-      "uuid": "80ae6a8e-8464-4f3a-afed-14d424757413",
-      "slug": "allergies",
       "core": false,
-      "unlocked_by": null,
       "difficulty": 3,
+      "slug": "allergies",
       "topics": [
-        "bitwise operations",
+        "bitwise_operations",
         "filtering"
-      ]
+      ],
+      "unlocked_by": null,
+      "uuid": "80ae6a8e-8464-4f3a-afed-14d424757413"
     },
     {
-      "uuid": "ba4e0186-0389-af80-d3e4-695dfc4e63bff1fa503",
-      "slug": "atbash-cipher",
       "core": false,
-      "unlocked_by": null,
       "difficulty": 3,
+      "slug": "atbash-cipher",
       "topics": [
-        "strings",
-        "algorithms"
-      ]
+        "algorithms",
+        "strings"
+      ],
+      "unlocked_by": null,
+      "uuid": "ba4e0186-0389-af80-d3e4-695dfc4e63bff1fa503"
     },
     {
-      "uuid": "ce9fd8ac-0beb-1a80-4ff6-be19ce1540d905c31b7",
-      "slug": "bracket-push",
       "core": false,
-      "unlocked_by": null,
       "difficulty": 3,
+      "slug": "bracket-push",
       "topics": [
-        "pattern matching",
+        "pattern_matching",
         "stacks"
-      ]
+      ],
+      "unlocked_by": null,
+      "uuid": "ce9fd8ac-0beb-1a80-4ff6-be19ce1540d905c31b7"
+    },
+    {
+      "difficulty": 2,
+      "slug": "all-your-base",
+      "topics": null,
```

Comment:
Please add some topics here. The list is at https://github.com/exercism/problem-specifications/blob/master/TOPICS.txt